src/PSR-12/CodeStyle.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
:warning: **Warning**: A file should declare new symbols (classes, functions, constants, etc.) and cause no other side effects, or it should execute logic with side effects, but should not do both. The first symbol is defined on line 36 and the first side effect is on line 42 (*PSR1.Files.SideEffects.FoundWithSymbols*).